beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -1274,6 +1280,9 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
                     RPCResponse::BlocksByRoot(resp) => {
                         self.build_response(id, peer_id, Response::BlocksByRoot(Some(resp)))
                     }
+                    RPCResponse::LightClientBootstrap(bootstrap) => {
+                        self.build_response(id, peer_id, Response::LightClientBootstrap(bootstrap))
```

Comment:
This is probably the one that's unreachable in practice because we never send light client bootstrap requests. But seeing as this code typechecks we may as well leave it here rather than putting another `unreachable!`